templates/zerver/help/set-notifications-for-a-single-stream.md

```diff
@@ -0,0 +1,20 @@
+# Set notifications for a single stream
+
+If you would like to have different notifications for different streams,
```

Comment:
What do you mean "different notifications for different streams"? This sounds kind of awkward.